src/PTY.ts

```diff
@@ -1,32 +1,48 @@
 import * as ChildProcess from "child_process";
 import * as OS from "os";
 import * as _ from "lodash";
-import * as pty from "pty.js";
+import * as pty from "node-pty";
 import {loginShell} from "./utils/Shell";
 import {debug} from "./utils/Common";
 
+let smth = true ? null : pty.createTerminal(undefined, undefined, undefined);
+type ITerminal = typeof smth;
```

Comment:
I had to do that hack because:
a) `node-pty` doesn't export its `ITerminal` type
b) https://github.com/Microsoft/TypeScript/issues/6606 isn't implemented in TypeScript (so we cannot do `type ITerminal = typeof(pty.createTerminal(undefined, undefined, undefined))`

Both could be solved by either exporting the proper types from `node-pty` library or writing our own type definitions (it's not that complex).